frontend/src/pages/chat/components/chat-footer/chat-footer.tsx

```diff
@@ -19,6 +19,10 @@ const ChatFooter: React.FC<Properties> = ({ onSend }) => {
 
   const onSubmit = useCallback(
     ({ text }: ChatInputValue): void => {
+      if (!text.trim()) {
+        return;
+      }
```

Comment:
this should be handled by validation schema, we will add this in chat messaging issue #190
```suggestion
```

Reply:
I understand, just made it for temporal use, till it is not implemented. Ok, I will remove it